docs/source/security/cors.mdx

```diff
@@ -84,108 +82,135 @@ If you create a public API or an API to embed in websites you don't control your
+
+## Configuring CORS options for Apollo Server
 
-##  Configuring CORS options for Apollo Server
+> 📣 **New in Apollo Server 4**: if you are using an Apollo Server integration (e.g., `expressMiddleware`), you are responsible for [setting up CORS for your web framework](../migration#body-parser-and-cors).
 
-By default, the batteries-included `apollo-server` package serves the `Access-Control-Allow-Origin` HTTP header with the wildcard value (`*`). This allows scripts on any origin to make requests, _without cookies_, to the server and read its responses.
+By default, Apollo Server's standalone server (i.e., `startStandaloneServer`) serves the `Access-Control-Allow-Origin` HTTP header with the wildcard value (`*`). This allows scripts on any origin to make requests, _without cookies_, to the server and read its responses. 
 
 > If you need to pass credentials to your server (e.g., via cookies), you can't use the wildcard value (`*`) for your origin. You _must_ provide specific origins. For examples, see [Passing credentials with CORS](#passing-credentials-with-cors).
 
-The batteries-included `apollo-server` [has a `cors` option](../api/apollo-server/#cors), which you can use to change your server's default CORS behavior:
-
-```js
-const server = new ApolloServer({
-  typeDefs,
-  resolvers,
-  csrfPrevention: true,  // see below for more about this
-  cache: "bounded",
-  cors: {
-    origin: ["https://www.your-app.example", "https://studio.apollographql.com"]
-  },
-  plugins: [
-    ApolloServerPluginLandingPageLocalDefault({ embed: true }),
-  ],
-});
+The [`startStandaloneServer` function](../migration#migrate-from-apollo-server) **doesn't support** configuring your server's CORS behavior.  If you want to customize your server's CORS behavior (e.g., by specifying origins or passing cookies), swap to using [`expressMiddleware`](../migration#migrate-from-apollo-server-express) (or any other Apollo Server integration). 
+
+Below, we set up and customize the CORS behavior for our `expressMiddleware` function using the [`cors` package](https://github.com/expressjs/cors#configuration-options):
+
+<MultiCodeBlock>
+
+```ts
+import { ApolloServer } from '@apollo/server';
+import { expressMiddleware } from '@apollo/server/express4';
+import { ApolloServerPluginDrainHttpServer } from '@apollo/server/plugin/drainHttpServer';
+import express from 'express';
+import http from 'http';
+import { json } from 'body-parser';
+import { typeDefs, resolvers } from './schema';
+// highlight-start
+import cors, { corsOptions } from 'cors';
+// highlight-end
+
+async function startApolloServer() {
+  const app = express();
+  const httpServer = http.createServer(app);
+  const server = new ApolloServer({
+    typeDefs,
+    resolvers,
+    plugins: [ApolloServerPluginDrainHttpServer({ httpServer })],
+  });
+
+  await server.start();
+
+  // highlight-start
+  const corsOptions: corsOptions = {
```

Comment:
Is it helpful to have a separate const vs just writing the options in the call? (FWIW the type name should be capitalized I think.)

Reply:
Good point, they separated out the options in the `cors` docs, but I agree it looks cleaner to provide the options straight to `cors` (and then I can omit explicitly typing the options too)!

```ts
  app.use(
    '/graphql',
    // highlight-start
    cors<cors.CorsRequest>({ origin: ['https://www.your-app.example', 'https://studio.apollographql.com'] }),
    // highlight-end
    json(),
    expressMiddleware(server),
  );
```